ansible/roles/automation/nightly_pr/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+- name: schedule opening nightly PRs
+  cron:
+    name: "openclose_{{ item.name }}"
+    minute: "{{ item.minute }}"
+    hour: "{{ item.hour }}"
+    weekday: "{{ item.weekdays }}"
+    flow: "{{ item.flow }}"
+    job: >
+           python3 {{ basedir }}/open_close_pr.py
+           open_nightly_pr
+           --config {{ basedir }}/config.yml
+           --id {{ item.name }}
+           --repo_path {{ basedir }}/freeipa
+           --flow {{ flow }}
+           --branch {{ branch }}
```

Comment:
The branch var is set by calling the top playbook prepare_openclose_pr_tool.yml, which means it has the same value for all the nightly job items (=master). It may not work well for setting the nightly PR for ipa-4-7 branch.